pyproject.toml

```diff
@@ -28,10 +28,13 @@ homepage = "https://github.com/exasol/mcp-server"
 fastmcp = "^2.10.4"
 pyexasol = "^0.27.0"
 sqlglot = "^27.2.0"
+spacy = "<3.8.7"
+pip = "^25.2"
```

Comment:
That could get problematic if we want to compile the server into a executable

Reply:
Hmm, yes it's a pain. Hopefully we will be able to find some solution.